src/components/textarea/bl-textarea.test.ts

```diff
@@ -30,7 +30,7 @@ describe("bl-textarea", () => {
           <div class="hint"></div>
         </div>
       `,
-      { ignoreAttributes: ["for", "id"] }
+      { ignoreAttributes: ["for", "id", "style"] }
```

Comment:
what is it for?

Reply:
Different browsers interpret styles differently like margin:4px or margin: 4px. This inconsistency could cause error for the test. It came with the latest version of Playwright.